README.md

```diff
@@ -1,2 +1,77 @@
-# URL Shortner
-URL Shortner is a Simple Web App Tool, which is used to shorten long URLs.
+<h3 align="center">URL Shortner</h3>
+
+  <p align="center">
+    URL Shortner is a Simple Web App Tool, which is used to shorten long URLs.   
+</p>
+
+
+
+<!-- TABLE OF CONTENTS -->
+## Table of Contents
+
+* [URL Shortner](#about-the-project)
+  * [Built With](#built-with)
+* [Getting Started](#getting-started)
+  * [Prerequisites](#prerequisites)
+  * [Installation](#installation)
+* [Contributing](#contributing)
+
+
+
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+
+Tis project is about to shortner the URL with the help of python. 
+
+### Built With
+This project is build with following languages and framework
+* [Python](https:://docs.python.org/release/3.9.7/)
+* [JavaScript](https:://www.javascript.com/)
+* [HTML](https:://html.com)
+* [CSS](https:://developer.mozilla.org/en-US/docs/Web/CSS)
+
+
+
+<!-- GETTING STARTED -->
+## Getting Started
+
+This is an example of how you may give instructions on setting up your project locally.
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+* Python
+
+*  Text editor
+
+### Installation
+1. Fork the repo in your github account.
+
+2. Clone the repo
+```sh
+git clone https:://github.com/<your_username_>/URL-Shortner.git
+```
+3. Move toward the Root directory .
+```sh
+   cd <Directory name>
+```
+4. Install packages
+```sh
+ pip install -r requirements.txt
+```
+5. Command for starting Flask Server
+```sh
+    set FLASK_APP=hello
```

Comment:
since we've Installed all required modules to start a Flask development Server, It can be started with by simply executing `main.py` file
So you can use 
```sh
python main.py
```

Reply:
okay got it